tests/migration_test.go

```diff
@@ -2746,17 +2779,11 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					_ = tests.RunMigration(virtClient, migration)
 
 					By("Expecting for an alert to be triggered")
-					Eventually(func() []k8sv1.Event {
```

Comment:
Wouldn't it be enough to check it is attached to the VMI?

Reply:
What do you mean exactly?